src/map/atcommand.c

```diff
@@ -7720,11 +7720,19 @@ ACMD(iteminfo)
 	for (i = 0; i < count; i++) {
 		struct item_data *item_data = item_array[i];
 		if (item_data != NULL) {
+
+			struct item link_item = { 0 };
+			link_item.nameid = item_data->nameid;
+			StringBuf buf;
+			StrBuf->Init(&buf);
+			clif->format_itemlink(&buf, &link_item);
+
 			snprintf(atcmd_output, sizeof(atcmd_output), msg_fd(fd, MSGTBL_ITEMINFO_DETAILS), // Item: '%s'/'%s'[%d] (%d) Type: %s | Extra Effect: %s
-				item_data->name, item_data->jname, item_data->slot, item_data->nameid,
+				item_data->name, StrBuf->Value(&buf), item_data->nameid,
```

Comment:
This is reducing the amount of arguments passed to snprintf, but the actual format string hasn't been updated, resulting in errors, crashes or string glitches at runtime.

You can find the string in `conf/messages.conf`, entry id 1277. Please also update the format string in the comment here and the one in the comment in `src/common/msgtable.h` (search `MSGTBL_ITEMINFO_DETAILS`), so that they are all identical

Reply:
This PR came from a diff from my personal changes and I completely missed this, my bad.
This is now fixed.